src/web/mjs/connectors/Manatoki.mjs

```diff
@@ -0,0 +1,59 @@
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
+
+export default class Manatoki extends GnuBoard5BootstrapBasic2 {
+
+    constructor() {
+        super();
+        super.id = 'manatoki82';
+        super.label = 'Manatoki';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://manatoki82.net';
+
+        this.path = '/comic/p%PAGE%';
+        this.pathMatch = /comic\/p(\d+)/;
+        this.queryMangasPageCount = 'div.list-page ul.pagination li:last-child a';
+        this.queryMangas = 'ul#webtoon-list-all li div.img-item div.in-lable a';
+        this.queryChapter = 'div.serial-list li.list-item div.wr-subject a';
+        this.scriptPages = `
+        new Promise(resolve => {
+            resolve([...document.querySelectorAll('div.view-padding div > img')].map(img => img.src));
+        });
+        `;
+
+    }
+    async _getChapters(manga) {
```

Comment:
What is the difference compared to the template?
Can the logic of this method merged into the template, so it is not required to overwrite this method?

Reply:
GnuBoard5BootstrapBasic2:
                id: this.getRootRelativeOrAbsoluteLink(element.dataset.role, this.url),

manatoki82:
                id: this.getRootRelativeOrAbsoluteLink(element, this.url),

Reply:
i don't think there is any way around this

Reply:
Maybe `id: this.getRootRelativeOrAbsoluteLink(element.dataset.role || element, this.url)` then